src/api/main.py

```
@@ -221,14 +234,67 @@ async def get_bus_punctuality(
+                inputs, config=config, stream_mode=["messages", "updates"]
+            ):
+                if await request.is_disconnected():
+                    return
+                for frame in project(mode, payload):
+                    if frame["type"] == "token":
+                        assistant_buffer.append(frame["content"])
+                    yield {"data": serialise(frame)}
+            yield {"data": serialise(frame_end())}
+        except asyncio.CancelledError:
+            raise
+        except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)